frontend/src/scenes/notebooks/Notebook/Notebook.tsx

```diff
@@ -99,9 +97,7 @@ export function Notebook({ shortId, editable = false, initialAutofocus = null }:
                 ) : null}
 
                 <div className="flex flex-1 justify-center space-x-2">
-                    <FlaggedFeature flag={FEATURE_FLAGS.NOTEBOOK_SETTINGS_WIDGETS}>
```

Comment:
Removing this because settings widgets are much more stable (and necessary now) given insights are working